teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(name, age, specialization)
+    @specialization = specialization
+    super(name, age)
+  end
```

Comment:
- [ ] Kindly arrange the `#super` parameters correctly.

- [ ] The `name` should be optional.

  ```
  def initialize(age, specialization, name = 'Unknown')
    super(age, name)
    @specialization = specialization
  end
  ```